charts/instana-agent/v1.0.15/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+name: instana-agent
+version: 1.0.15
+appVersion: 1.0
+description: Instana Agent for Kubernetes
+home: https://www.instana.com/
+icon: http://instana-management-assets.s3-website-eu-west-1.amazonaws.com/stan_icon_front_black_big.png
```

Comment:
would prefer to add the logo to the parent directory and use it via `icon: file://../logo.png` for air-gapped users.